### PR TITLE
ZD-857: FTS: directives support on file upload

### PR DESCRIPTION
**Pull request by PavelLoparev**

### CODE REVIEW

File: api/file-translations/params/fts-upload-file-parameters.ts [hunk 1]
```diff
@@ -26,4 +26,17 @@ export class FtsUploadFileParameters extends BaseParameters {
         this.parameters.request.fileType = fileType.toUpperCase();
         return this;
     }
+
+    addDirective(instruction: string, value: string): FtsUploadFileParameters {
+        if (!this.parameters.request.parseConfigItems) {
+            this.parameters.request.parseConfigItems = [];
+        }
```
Comment (PavelLoparev) [inline api/file-translations/params/fts-upload-file-parameters.ts:33]: This is how directives are passed to FTS while file upload

File: test/file-translations.spec.ts [hunk 2]
```diff
@@ -105,7 +105,9 @@ describe("SmartlingFileTranslationsApi class tests.", () => {
 
             params
                 .setFileType(FileType.XML)
-                .setFileContent(testFileContent);
+                .setFileContent(testFileContent)
+                .addDirective("entity_escaping_strategy", "propagade")
```
Comment (dimitrystd) [inline test/file-translations.spec.ts:109]: I believe all directives should have `smartling.` prefix. I understand that we can put whatever we want in the test. But some times tests are a good source to figure out how it should work.

**Review by dimitrystd — APPROVED** [review]

File: test/file-translations.spec.ts (hunk 2 above)
Comment (PavelLoparev) [reply test/file-translations.spec.ts:109]: No. Exactly in FTS there is a validation that doesn't allow `smartling.<...>` format. COnfirmed with @andy-radchenko

**Review by vfisyuk-smartling — APPROVED** [review]

**Comment by PavelLoparev** [comment]
I reverted this change in this sdk and released `2.13.0`. FTS directives support is incomplete on FTS side and is not public. I will implement this in private sdk

**Pull request merged**